.travis.yml

```diff
@@ -8,43 +8,52 @@ dist: focal
 language: python
 jobs:
   include:
-    - name: “Python 2.7 on xenial”
-      python: "2.7"
+    - name: Python 2.7 on xenial
+      python: 2.7
       dist: xenial
-    - name: “Python 2.7 on focal”
-      python: "2.7"
-    - name: “Python 2.7 on Infogami master”
-      python: "2.7"
+    - name: Python 2.7 on focal
+      python: 2.7
+    - name: Python 2.7 on Infogami master
+      python: 2.7
       script:
         - make i18n
-        - make test
+        - make test-py
         - pip install safety
         - safety check || true
-    - name: “Python 3.8 on focal”
-      python: "3.8"
-    - name: “Python 3.9-dev on focal”
-      python: "3.9-dev"
+    - name: Python 3.8 on focal
+      python: 3.8
+    - name: Python 3.9-dev on focal
```

Comment:
Please revert the removal of quotes on floating point values on yaml data.  Next week Python 3.9 ships.  The release next year at this time will be 3.10.  Without quotes on floating point numbers in yaml 3.10 is indispensable from 3.1 and Python 3.10 will be less than, not greater than 3.9.
```python
yaml_data = """
three_one: 3.1
three_eight: 3.8
three_nine: 3.9
three_ten: 3.10
"""
import yaml
print(yaml.load(yaml_data))
```

Reply:
Ahh, good point! Fixing

Reply:
Done.